From: Michael Davidsaver <mdavidsaver@gmail.com>
Date: Sat, 23 Jun 2018 14:48:13 -0700
Subject: fix EPICSMake.cpp path

---
 cmake/Modules/FindEPICSTools.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmake/Modules/FindEPICSTools.cmake b/cmake/Modules/FindEPICSTools.cmake
index 11868ea..660221e 100644
--- a/cmake/Modules/FindEPICSTools.cmake
+++ b/cmake/Modules/FindEPICSTools.cmake
@@ -42,7 +42,7 @@ foreach(TOOL ${EPICS_TOOLS})
 endforeach()
 
 find_file(EPICS_MAIN_TEMPLATE EPICSMain.cpp
-  PATHS ${CMAKE_MODULE_PATH}/../Templates
+  PATHS ${CMAKE_CURRENT_LIST_DIR}/../Templates
   NO_DEFAULT_PATH
   NO_CMAKE_SYSTEM_PATH
   NO_CMAKE_FIND_ROOT_PATH
