From: Michael Davidsaver <mdavidsaver@gmail.com>
Date: Thu, 26 Jul 2018 12:12:06 -0700
Subject: extend default module search path

---
 cmake/Modules/FindEPICSModule.cmake | 1 +
 1 file changed, 1 insertion(+)

diff --git a/cmake/Modules/FindEPICSModule.cmake b/cmake/Modules/FindEPICSModule.cmake
index d2d62d6..aaef271 100644
--- a/cmake/Modules/FindEPICSModule.cmake
+++ b/cmake/Modules/FindEPICSModule.cmake
@@ -75,6 +75,7 @@ function(find_epics_module)
       ${EPICS_MODULE_PATH}
       ${FEM_PATH}
       ${EPICS_BASE_DIR}/../${FEM_NAME}
+      ${EPICS_BASE_DIR}/../modules/${FEM_NAME}
       ${EPICS_BASE_DIR}
     NO_DEFAULT_PATH
     NO_CMAKE_SYSTEM_PATH
